### PR TITLE
Add missing items to the response body in guides/bulk.md

### PR DESCRIPTION
**Pull request by ap-h**
### Description

_Describe what this change achieves._

### Issues Resolved

_List any issues this PR will resolve, e.g. Closes [...]._

### Check List

- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] Linter check was successfull - `yarn run lint` doesn't show any errors
- [ ] Commits are signed per the DCO using --signoff
- [ ] Changelog was updated.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: guides/bulk.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ client.bulk({
     { create: { _id: 2, data: { title: 'Beauty and the Beast 4', year: 2049 } } }  // document already exists error
   ]
 }).then((response) => {
-  response.body.forEach((item) => {
+  response.body.items.forEach((item) => {
```
Comment (ap-h) [inline guides/bulk.md:150]: Is it possible `items` is undefined or null?

**Review by dblock — APPROVED** [review]

**Comment by ap-h** [comment]
> We had similar problems in guides, so we started adding working samples into https://github.com/opensearch-project/opensearch-js/tree/main/samples. Do you want to extract this code into a working sample and reference it? This way we know the code works 100%.

Thanks for the review, yep I think I can do this over the weekend, do you know any example how samples added to the guides? Just for wordings

**Comment by dblock** [comment]
> > We had similar problems in guides, so we started adding working samples into https://github.com/opensearch-project/opensearch-js/tree/main/samples. Do you want to extract this code into a working sample and reference it? This way we know the code works 100%.
> 
> Thanks for the review, yep I think I can do this over the weekend, do you know any example how samples added to the guides? Just for wordings

It's all over the place, e.g. https://github.com/opensearch-project/opensearch-js/blob/4e615f3d38bfc149fd94d3009ca824b4f6a3b862/guides/search.md#search references a sample file. I am happy with anything that links to it.

Someone will go edit all the docs, write all the samples, and make it awesome. Wink wink :)

**Comment by nhtruong** [comment]
The guide has been reworked. Please check if this change is still needed anymore or if you wanna make other updates to it :)

**Comment by ap-h** [comment]
Thanks @nhtruong, I don't think this is needed anymore.

**Pull request closed without merging**